src/components/Tooltip/Tooltip.stories.mdx

```diff
@@ -18,72 +20,159 @@ import { FIGMA_URL } from '../../utils/common';
 
 ## Props
 
 <Props of={Tooltip} />
 
-### Generic Tooltip
+## Usage
+
+<UsageGuidelines
+  guidelines={[
+    'To display the full string of a truncated text',
+    'To provide additional context for unlabeled icon buttons',
+    'Use text tooltips to offer additional context that does not support links or actions',
+    'Use interactive tooltips for more complex layouts and to offer simple actions within the tooltip',
+    'Do not display critical information in a tooltip',
+    'Do not use tooltips to display information unrelated to the triggered element',
+  ]}
+/>
+
+## Variants
+
+### Text Tooltip
+
+Text Tooltip takes a string as a content prop. There are 2 variations of Text Tooltips, default and inverted.
+
+<Preview>
+  <Story name="Text Tooltip">
+    <Stack>
+      <Function>
+        {() => {
+          return (
+            <div style={{ padding: '16px' }}>
+              <Tooltip content={'This is a Tooltip'} placement={'right'}>
+                <Button>Hover here</Button>
+              </Tooltip>
+            </div>
+          );
+        }}
+      </Function>
+    </Stack>
+    <Stack isInverted>
+      <Function>
+        {() => {
+          return (
+            <div style={{ padding: '16px' }}>
+              <Tooltip content={'This is a Tooltip'} placement={'right'} isInverted>
+                <Button type="inverted">Hover here</Button>
+              </Tooltip>
+            </div>
+          );
+        }}
+      </Function>
+    </Stack>
+  </Story>
+</Preview>
+
+### Interactive Tooltip
 
-Tooltip use to show info in any direction defined. If there is no space in the screen then it moves to the available space.
+Interactive Tooltip takes a React Component as a content prop. There are 2 variations of Text Tooltips, default and inverted.
 
 <Preview>
-  <Story name="Generic Tooltip" parameters={{ decorators: [withKnobs] }}>
-    <TooltipShowcase
-      content={text(
-        'content',
-        'By hovering here you can see very long and useful info that totally breaks into two lines'
-      )}
-    />
+  <Story name="Interactive Tooltip">
+    <Stack>
+      <TooltipShowcase />
+    </Stack>
+    <Stack isInverted>
+      <TooltipShowcase isInverted />
+    </Stack>
   </Story>
 </Preview>
 
-### Tooltip sizes
+### Tooltip Placement
+
+Tooltip can be placed to the top, bottom, left or right of the element.
+
+<Preview>
+  <Story name="Tooltip Placement" parameters={{ decorators: [withKnobs] }}>
+    <Stack>
+      <Function>
+        {() => {
+          return (
+            <div
+              style={{
+                padding: '16px',
+                width: '300px',
+                height: '100px',
+                display: 'flex',
+                justifyContent: 'center',
+                alignItems: 'center',
+              }}
+            >
+              <Tooltip
+                content={'This is a Tooltip'}
+                placement={select('placement', ['top', 'right', 'bottom', 'left'], 'right')}
```

Comment:
in this story i would expect all 4 placements as showcase with no playground on this property